### ISSUE TITLE
Games do not work

### ISSUE THREAD

**Issue by airtag8892** [3kh0/3kh0.github.io#92]
### Check these boxes to make sure you did everything right!

- [X] I have made sure that this issue does not already exist
- [X] I have submitted screenshots of the error
- [X] You are not using the website on any clones (you are using 3kh0.github.io and nothing else)
- [ ] I just checked all of these boxes without reading the labels

### What browser are you using?

Other

### If you said other above, please specify which browser you use

Tested on Safari and Google Chrome

### What device are you using?

Other (please note in text field below)

### If you said other above, please specify which device you use

Chromebook, iPhone 11

### Please explain the issue

Most if not all games currently do not work and redirects to a 404 error. (someone else tested it on a chromebook and it also didn’t work)

Requested URL: https://3kh0.github.io/files/2048/index.html
Referrer: https://3kh0.github.io/projects
Title: Error | 3kh0
Screen height: 896px
Screen width: 414px
Window height: 715px
Window width: 414px
Cookies: true
User agent: Mozilla/5.0 (iPhone; CPU iPhone OS 16_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.3 Mobile/15E148 Safari/604.1
OS platform: iPhone
Online: true
Error code: 404

**Comment by idkaname1** [comment]
That was my problem

**Comment by idkaname1** [comment]
![image](https://user-images.githubusercontent.com/125160132/218282738-37baa0b5-fe87-4bb0-82aa-3b51951d4e87.png)

**Comment by Russell2259** [comment]
That might be me...

Try doing ctrl+shift+r or ctrl+reload.

#83

**Comment by idkaname1** [comment]
no

**Comment by idkaname1** [comment]
it didn't work

**Comment by Russell2259** [comment]
@3kh0 a patch for this issue is in pr #90

**Comment by Russell2259** [comment]
> it didn't work

Here is a temporary fix:

Click on the lock in the url bar.
Then click site settings.
After that page loads click clear data.
Reload 3kh0.github.io

**Comment by idkaname1** [comment]
tysm

**Comment by idkaname1** [comment]
i love you like a brother

**Comment by idkaname1** [comment]
ps: could you possibly add bonk.io

**Comment by idkaname1** [comment]
it breaks if i open up singleplayer in eaglerccraft 1.5

**Comment by Russell2259** [comment]
hmm... I'll look into it. I pushed a temporary patch but I'm waiting for @3kh0 to merge it.

**Comment by somethingishere** [comment]
it keeps telling me that this page dose not exist

**Comment by Russell2259** [comment]
Click on the lock in the url bar.
Then click site settings.
After that page loads click clear data.
Reload 3kh0.github.io

I'm waiting on 3kh0 to merge my push request. Hopefully it works like normal tomorrow.

**Comment by Russell2259** [comment]
Read issue #93 for a fix.

**Comment by 3kh0** [comment]
@airtag8892 it should be fixed. try now. if it does not work, make a new issue
